mpls/draft-ietf-detnet-mpls.xml

```diff
@@ -110,13 +110,21 @@
 	Architecture <xref target="RFC8655"/>.
   </t>
   <t>
-    The DetNet Architecture models the DetNet related data plane functions
-    decomposed
-    into two sub-layers: a service sub-layer and a forwarding sub-layer.  The
-    service sub-layer is used to provide DetNet service functions such as
-    protection and reordering.  The forwarding sub-layer is used to provide
-    forwarding assurance (low loss, assured latency, and limited out-of-order
-	delivery).
+   The purpose of this document is to describe the use of the MPLS 
+   data plane to establish and support DetNet flows.
+   The DetNet Architecture models the DetNet related data plane
+   functions decomposed into two sub-layers: a service sub-layer and a
+   forwarding sub-layer.  The service sub-layer is used to provide
+   DetNet service functions such as protection and reordering.  At the
+   DetNet data plane a new set of functions (PREOF) provide the service
+   sub-layer specific tasks. The forwarding sub-layer is used to provide
+   forwarding assurance (low loss, assured latency, and limited
+   out-of-order delivery) via already existing Traffic Engineering
+   and queuing tools. The use of the functionalities of the DetNet 

```

Comment:
New tools (queuing mechanisms, TSpecs are also allowed", so I'd prefer to clarify or remove "via already existing Traffic Engineering  and queuing tools."